### DISCUSSION TITLE
Kernel dies when reading UCN file on Mac

### DISCUSSION THREAD

**Discussion by mbakker7** [modflowpy/flopy#1991] [General]
I am working on a Mac.
I am running MT3D and get a UCN file. 
mf2005 and MT3D run fine.
I then create a UCN object and can get the times out of the UCN file (with the `get_times()` function). All good so far.
When I try to read all data (with the `get_alldata()` function), the Kernel dies.

Now get this: Running the exact same script on a Windows machine works fine.

Any thoughts where I should start looking for an error?
Incidentally, I have run the `mt3d_1d` example from the flopy website, and that works fine on my Mac.

**Comment by christianlangevin** [comment]
Presumably, the binary executables came from here?  https://github.com/MODFLOW-USGS/executables

**Reply by mbakker7** [reply]
They sure did. And the `mt3d_1d` example works fine. I may start to modify this file with my input and see where the problems start occurring.
